docs/java-rest/high-level/supported-apis.asciidoc

```diff
@@ -95,6 +95,7 @@ include::indices/clear_cache.asciidoc[]
 include::indices/force_merge.asciidoc[]
 include::indices/rollover.asciidoc[]
 include::indices/put_mapping.asciidoc[]
+include::indices/get_mappings.asciidoc[]
```

Comment:
heya @dakrone this is not enough for the page to be linked, in fact this API is currently missing in our docs, would you mind fixing that please?